ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__aws-4.20-nightly-x86-loaded-upgrade-from-4.19.yaml

```diff
@@ -0,0 +1,235 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  initial:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.19.0-0
+        upper: 4.20.0-0
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade-419to420-24nodes
```

Comment:
These generated configs won't have slack reporter configs because they have a new config name in this file. To do that, please add all the net-new names from this file to the .config.prowgen, like these:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/.config.prowgen#L35-L37